core/pythonAction/Dockerfile

```diff
@@ -8,24 +8,23 @@ RUN apk add --no-cache \
         libxslt-dev \
         libxml2-dev \
         libffi-dev \
-        openssl-dev \
-        python-dev
+        openssl-dev
 
 # Install common modules for python
 RUN pip install \
-    beautifulsoup4==4.5.1 \
-    httplib2==0.9.2 \
-    kafka_python==1.3.1 \
-    lxml==3.6.4 \
-    python-dateutil==2.5.3 \
-    requests==2.11.1 \
-    scrapy==1.1.2 \
-    simplejson==3.8.2 \
-    twisted==16.4.0
```

Comment:
need to update these versions https://github.com/openwhisk/openwhisk/blob/master/docs/reference.md#python-actions